Assignment-4/app/src/Components/FormValidationRule.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { FormControl, TextField, FormGroup, Button } from "@mui/material";
+import { useFormik } from "formik";
+import * as yup from "yup";
+function FormValidationRule() {
+  const [formFields, setFormFields] = useState({
+    email: "",
+    password: "",
+    phoneno: "",
+  });
+
+  const validationSchema = yup.object({
+      password: yup
+      .string()
+      .min(7, " minimum password letter should be 7 !")
+      .max(15,'inimum password letter should be 7 !').required("Password is required"),
+    phoneno: yup
+      .string()
+      .min(10, "mobile no is not valid")
+      .max(10, "mobile no is not valid!")
+      .required("mobile no is Required"),
+    email: yup.string().email("Invalid email").required("Email is Required"),
+  });
+
+  const handleSubmit = () => {
+    console.log(formFields);
+  };
+
```

Comment:
![image](https://github.com/kaushalyad321/ReactJs/assets/83696224/5ee2669c-ce07-43a2-af88-bf8486cbdebf)
email validation is not correct

Reply:
ok fixing